output_frame: 'lidarlite'
altitude_variance: 0.0009
max_altitude: 4.0
